enclave/src/tests.rs

```diff
@@ -138,6 +139,7 @@ pub extern "C" fn test_main_entrance() -> size_t {
         test_proxy::test_add_proxy_account,
         test_proxy::test_remove_proxy_account,
         test_proxy::test_remove_main_account,
+		happy_path::test_happy_path,
```

Comment:
This should probably be moved to integration tests, not unit tests. But I'll mention in issue #152

Reply:
So no need to move this for this PR.

Reply:
I am sorry .. I didnt understand this ... do you want me to remove it?

Reply:
No, just leave it.

Reply:
oh .. okay .. thanks